docs/GlusterFS-Tools/glusterfind.md

```diff
@@ -96,54 +108,54 @@ Examples,
 
 Output file contains list of files/dirs relative to the Volume mount, if we need to prefix with any path to have absolute path then,
 
-```console
+```{ .console .no-copy }
 # glusterfind pre backup datavol /root/backup.txt --file-prefix=/mnt/datavol/
 ```
 
 ### List Command
 
 To get the list of sessions and respective session time,
 
-```console
+```{ .console .no-copy }
 glusterfind list [--session SESSION_NAME] [--volume VOLUME_NAME]
 ```
 
 Examples,
 
-```console
+```{ .console .no-copy }
 # glusterfind list
 # glusterfind list --session backup
 ```
 
 Example output,
 
-```console
+```{ .text .no-copy }
 SESSION                   VOLUME                    SESSION TIME
 ---------------------------------------------------------------------------
 backup                    datavol                   2015-03-04 17:35:34
 ```
 
 ### Post Command
 
-```console
+```{ .console .no-copy }
```

Comment:
no-copy not required

Reply:
Actually the no-copy attr is only omitted from places where the code can be copied and pasted and it will run. Like in the examples below. This part just dictates the syntax and hence copy is disabled. I enabled the copy button for example code blocks below this.